### DISCUSSION TITLE
Initial starting-point (date) to select a date in DateRangePicker

### DISCUSSION THREAD

**Discussion by michaelhuber1937** [mantinedev/mantine#1677] [Q&A] [answered]
Let's say, I have set min/max date to 01/01/2017 - 12/31/2017 because the selected date must be in that range. The initial value for this DateRangePicker is [] because it works like a filter for a huge dataset.

When the user now clicks the DateRangePicker, the current date is used to display the calendar, so the user has to click backwards to 2017 to select a valid date, a lot of clicks to go back to 2017 which is annoying UX.

So is there an option to define a date which is used to present the inital month being displayed? In my case that would be January 01, 2017 as a starting point for the user to select his desired range.

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/cranky-minsky-iilvu8?file=/src/App.tsx

**Reply by michaelhuber1937** [reply]
Thanks a lot. I searched for the appropriate property but could not find it by myself. Works like a charm.
